main.js

```diff
@@ -0,0 +1,72 @@
+//Make a for loop that logs outevennumbers from 10- 40.
+//However, if the number is divisible by 3 it should log "Divisible by 3!".
+//If it's divisible by 5 it should log "Divisible by5!"
+//If it's divisible by both 3 and 5 it should log "Jackpot!"
+//Hint: you will need the modulo operator (%).
+//Here is an example of what should be logged:
+//10 "This is divisible by 5"
+//12 "This is divisible by 3
+
+
+const numbers=[10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40]; 
+
+for(let i = 10; i<= 40; i++ ){
+
+      if (i % 3=== 0){
+        console.log('Divisible by 3'); 
+
+     }
+     else if(i % 5 === 0){
+        console.log('Divisible by 5')
+     }
+        else if (i % 3 === 0 && i % 5 === 0){
+            console.log('Jackpot'); 
+        }
+        else {console.log(i); 
+        } 
+ 
+}; 
```

Comment:
Well done! The logic is good, but let me note that the jackpot `if` should be the first, otherwise, in the case of 15, your code will print `Divisible by 3Divisivle by 5Jackpot`, same in the case of 30. If you changed that, the logic would be perfect. Also, the output should contain the number, so it should be `i + 'Divisible by 3'` in the first `if` and in the other ones too. Great job on going up until 40, by using `<=` and not `<`. 💯